aws/cloudformation/lambdas/sql-user/index.js

```diff
@@ -0,0 +1,130 @@
+const mysql = require("mysql");
+const { sendCfnResponse } = require("./cfn-response");
+
+exports.handler = async (event, context) => {
+  // TODO: Mask password fields.
+  console.log("REQUEST RECEIVED:\n", JSON.stringify(event));
+
+  const props = event.ResourceProperties;
```

Comment:
Consider sanitizing these. This function is vulnerable to SQL Injection and could do more than create/update/delete users.